src/Persistence/MassTransit.MongoDbIntegration/Audit/AuditDocument.cs

```diff
@@ -0,0 +1,51 @@
+namespace MassTransit.MongoDbIntegration.Audit
+{
+    using System;
+    using MassTransit.Audit;
+    using Newtonsoft.Json;
+
+
+    public class AuditDocument
+    {
+        public string MessageId { get; set; }
```

Comment:
I'd like to have `private set`

Reply:
But why? Is there a reason to hide setters?

Reply:
IMHO, I always prefer to hide as much as possible